src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -3,14 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-# pylint: disable=too-few-public-methods,too-many-arguments,no-self-use,too-many-locals,line-too-long,unused-argument
+# pylint: disable=too-few-public-methods,too-many-arguments,no-self-use,too-many-locals,line-too-long,unused-argument,too-many-branches,too-many-statements
```

Comment:
Try to split the methods into smaller methods and simply the code flow before you add this two pylint disable.

Reply:
Please advise how. This method is called from `commands.py` which passes all the input parameters.

Reply:
Split function into the smaller units and make custom command method invoke the,